commands/generate-key.js

```diff
@@ -1,3 +1,4 @@
+const keyCurveTypes = require('near-api-js').keyCurveTypes;
```

Comment:
@TarasenkoFirelabs this line gives me undefined.

Reply:
I am still working onto this task in `near-api-js` repo

Reply:
Got it, then it's better to keep this one in a draft. Ping me when it will be ready for review, thanks!